{"uid":"4b6a0ab9426370c4","name":"Поиск в яндексе","fullName":"test.test_tensor.TensorTestCase#test_search","historyId":"6bc54fbf095856b02ce30f6849fa7bbe","time":{"start":1659260652268,"stop":1659260656011,"duration":3743},"status":"failed","statusMessage":"AssertionError: Открылась страница с капчей\nassert False","statusTrace":"self = <test.test_tensor.TensorTestCase testMethod=test_search>\n\n    def setUp(self):\n        self.browser_setup()\n>       self.navigate_to_home_page()\n\ntest/test_tensor.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test.test_tensor.TensorTestCase testMethod=test_search>\n\n    def navigate_to_home_page(self):\n        self.hp = Home_Page()\n        self.hp.navigate()\n        try:\n            assert Captcha_Page().form.checkbox.element.visible\n        except splinter.exceptions.ElementDoesNotExist:\n            pass\n        else:\n>           assert False, \"Открылась страница с капчей\"\nE           AssertionError: Открылась страница с капчей\nE           assert False\n\ntest/test_tensor.py:32: AssertionError","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TensorTestCase","time":{"start":1659260637313,"stop":1659260637313,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Открылась страница с капчей\nassert False","statusTrace":"self = <test.test_tensor.TensorTestCase testMethod=test_search>\n\n    def setUp(self):\n        self.browser_setup()\n>       self.navigate_to_home_page()\n\ntest/test_tensor.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test.test_tensor.TensorTestCase testMethod=test_search>\n\n    def navigate_to_home_page(self):\n        self.hp = Home_Page()\n        self.hp.navigate()\n        try:\n            assert Captcha_Page().form.checkbox.element.visible\n        except splinter.exceptions.ElementDoesNotExist:\n            pass\n        else:\n>           assert False, \"Открылась страница с капчей\"\nE           AssertionError: Открылась страница с капчей\nE           assert False\n\ntest/test_tensor.py:32: AssertionError","steps":[{"name":"Перейти на главную страницу Яндекса","time":{"start":1659260653315,"stop":1659260653315,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TensorTestCase::0","time":{"start":1659260656061,"stop":1659260656061,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"severity","value":"normal"},{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_tensor"},{"name":"subSuite","value":"TensorTestCase"},{"name":"host","value":"fv-az238-110"},{"name":"thread","value":"2968-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_tensor"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":4,"skipped":0,"passed":5,"unknown":0,"total":13},"items":[{"uid":"af5ff71f222815b7","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/45//#testresult/af5ff71f222815b7","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd447e6f280>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1659258678618,"stop":1659258678636,"duration":18}},{"uid":"cf36edd120614426","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/44//#testresult/cf36edd120614426","status":"passed","time":{"start":1659255982847,"stop":1659255993328,"duration":10481}},{"uid":"b8ad27141c1a2f2f","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/43//#testresult/b8ad27141c1a2f2f","status":"passed","time":{"start":1659206695241,"stop":1659206703371,"duration":8130}},{"uid":"97ddcfddabefef54","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/42//#testresult/97ddcfddabefef54","status":"passed","time":{"start":1659200003924,"stop":1659200011945,"duration":8021}},{"uid":"c04c0b21b2d69114","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/41//#testresult/c04c0b21b2d69114","status":"failed","statusDetails":"AssertionError: Открылась страница с капчей\nassert False","time":{"start":1659196944510,"stop":1659196948898,"duration":4388}},{"uid":"f15ecdbddc4c1521","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/39//#testresult/f15ecdbddc4c1521","status":"passed","time":{"start":1659194313648,"stop":1659194321808,"duration":8160}},{"uid":"ec1264e8fcc5b76d","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/38//#testresult/ec1264e8fcc5b76d","status":"failed","statusDetails":"AssertionError: Открылась страница с капчей\nassert False","time":{"start":1659193962930,"stop":1659193966576,"duration":3646}},{"uid":"526d38778751b92c","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/37//#testresult/526d38778751b92c","status":"passed","time":{"start":1659193828152,"stop":1659193837223,"duration":9071}},{"uid":"738c8b42a4d022d8","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/36//#testresult/738c8b42a4d022d8","status":"failed","statusDetails":"AssertionError: Открылась страница с капчей\nassert False","time":{"start":1659191734751,"stop":1659191738711,"duration":3960}},{"uid":"eac92800ab65ae10","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/34//#testresult/eac92800ab65ae10","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5722e09e40>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1659191418401,"stop":1659191418419,"duration":18}},{"uid":"a6f75d64911bcb2f","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/33//#testresult/a6f75d64911bcb2f","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fea7fb8bb20>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1659191268520,"stop":1659191268539,"duration":19}},{"uid":"3f611ceee3ba9edd","reportUrl":"https://schastev.github.io/2022_07_tensor_test_assignment/31//#testresult/3f611ceee3ba9edd","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7ddcbdfb80>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1659191009672,"stop":1659191009690,"duration":18}}]},"tags":[]},"source":"4b6a0ab9426370c4.json","parameterValues":[]}